spec/features/admin/budget_investments_spec.rb

```diff
@@ -588,7 +588,23 @@
         expect(page).to_not have_link('Selected')
       end
     end
+  end
+
+  context "Selecting csv" do
+
+    scenario "Downloading CSV file" do
+      investment1 = create(:budget_investment, :unfeasible, budget: @budget)
+      investment2 = create(:budget_investment, :feasible, budget: @budget)
 
+      visit admin_budget_budget_investments_path(@budget, format: :csv)
+
+      header = page.response_headers['Content-Disposition']
+      header.should match(/^attachment/)
+      header.should match(/filename="budget_investments.csv"$/)
+
+      expect(page).to have_content investment2.title
+      expect(page).to have_content investment1.title
```

Comment:
I like this way to check de csv download 👍  😃  I'm only missing a negative scenario, I mean: would it be possible to check that filtering by feasible only budget investments, the unfeasible ones doesn't appear at the file? Or any other scenario that you see easier to reproducer, where we can verify that only the investments that the user see are at the html view the ones exported in the file :)

Reply:
Ok, It will be in a moment